ipxe/files.go

```diff
@@ -0,0 +1,11 @@
+package ipxe
+
+import (
+	"embed"
+)
+
+//go:embed ipxe/undionly.kpxe
+//go:embed ipxe/ipxe.efi
+//go:embed ipxe/snp-nolacp.efi
+//go:embed ipxe/snp-hua.efi
```

Comment:
can you sort these please? Not important but does make me feel warm and fuzzy!

Reply:
Done!